src/generator/www/js/main.js

```diff
@@ -0,0 +1,52 @@
+$(document).ready(function() {
+
+  let $email =$("#email");
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [`$email` is never modified, use `const` instead.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711222409/issues/source?bid=3233857&fileBranchId=3344140#l3)